### DISCUSSION TITLE
Problems with my private Cloud

### DISCUSSION THREAD

**Discussion by sigsdis30** [opengisch/QField#4481] [Q&A]
Hi, 
We have implemented your QFieldCloud on our server. 
Our cloud runs on DEBIAN12 OS with a i5 processor, 16GB RAM and 500Go of disk space.  
The configuration has been mounted under DOCKER as specified in the GitHub page and we run the docker-compose.override.test.yml. 

I try using it but I can't publish a project onto the Cloud because "[inactive_subscription] Permission denied because the subscription is inactive".
I don't know how to make it work ... 

Do you have any user guide on how the personal Cloud works ? Or can you help us ?

**Comment by suricactus** [comment]
Please see https://github.com/opengisch/qfieldcloud/issues/668 .
